src/makeRobot.js

```diff
@@ -38,7 +38,68 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const rescueCenter = {
+    x: 1400,
+    y: 500,
+  };
+
+  const robot = {
```

Comment:
```suggestion
  return {
```
You can return everything right here

Reply:
Thanks for advice!